docs/commercial/single-sign-on.rst

```diff
@@ -0,0 +1,74 @@
+Single Sign-On
+==============
+
+.. note::
+
+   This feature only exists on `Read the Docs for Business <https://readthedocs.com/>`__.
+
+
+Single Sign-On is supported on |com_brand| for Pro and Enterprise plans.
+:abbr:`SSO (Single Sign-On)` will allow you to manage all the users' permissions directly on GitHub.
+
+.. note::
+
+   Currently, we only supports GitHub as Identity Provider.
+   We plan to support GitLab and Bitbucket soon as well.
```

Comment:
I don't think either of these need to be notes, or we should try and keep only 1 of them, since a bunch of notes is sloppy.